packages/core/src/domain/configuration/tags.ts

```diff
@@ -0,0 +1,46 @@
+import { display } from '../../tools/display'
+import { InitConfiguration } from './configuration'
+
+export const TAG_SIZE_LIMIT = 200
+
+export function buildTags(configuration: InitConfiguration): string[] {
+  const { env, service, version } = configuration
+  const tags = []
+
+  if (env) {
+    tags.push(buildTag('env', env))
+  }
+  if (service) {
+    tags.push(buildTag('service', service))
+  }
+  if (version) {
+    tags.push(buildTag('version', version))
+  }
+
+  return tags
+}
+
+export function buildTag(name: string, value: string) {
```

Comment:
```suggestion
export function buildTag(key: string, value: string) {
```
in order to use the same naming as in the doc